data_structures/hashing/hash_table.py

```diff
@@ -173,7 +173,7 @@ def _set_value(self, key, data):
         self.values[key] = data
         self._keys[key] = data
 
-    def _collision_resolution(self, key, data=None):
+    def _collision_resolution(self, key, data=None):  # noqa: ARG002
```

Comment:
Same here?

Reply:
Yes. Done. `noqa` left in `data_structures/hashing/quadratic_probing.py` where this method is overridden